docs/collaboration/webhook.md

```diff
@@ -22,7 +22,7 @@ A sample payload of the webhook request looks like this:
 
 ### Signing
 
-All requests to your webhook URL will contain a header called `X-Hocuspocus-Signature-256` that signs the entire message with your secret. You can find it in the [settings](https://collab.tiptap.dev/apps/settings) of your Tiptap Collab app.
```

Comment:
HTTP headers are, out of convention, capitalized see MDN's docs: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

Your concern is around a Next.js implementation detail. The docs are more correct here